REFERENCE.md

```diff
@@ -436,14 +436,32 @@ Start charging the vehicle.
 
 ---
 
+### `send_destination(self)`
+
+Send destination to the vehicle.
+
+#### Return
```

Comment:
We need an Arguments table to show the parameters - latitude (Float), longitude (Float)